docs/src/catalyst_applications/simulation_structure_interfacing.md

```diff
@@ -11,35 +11,91 @@ using Catalyst
 rn = @reaction_network begin
     (k1,k2), X1 <--> X2
 end
+```
+
+To later enable us to index solutions and problems using symbolic variables we
+also need to indicate that we are finished constructing our system using `complete`:
+```@example ex1
+rn = complete(rn)
+nothing  # hide
```

Comment:
I don't think this belongs here, we should put a proper (but not long-winded) explanation of `complete` and similar somewhere. Then, we can reference that whenever we want to introduce complete systems.